### PR TITLE
HOCS-5000: stage duplication problem

### PR DESCRIPTION
**Pull request by gary-newbolt**

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java [hunk 1]
```diff
@@ -67,25 +67,17 @@ public String createStage(String caseUUIDString, String stageUUIDString, String
...
-
-            CreateCaseworkStageRequest stageRequest = new CreateCaseworkStageRequest(stageTypeString, teamUUID, userUUID, allocationType);
-            log.debug("Creating new stage {} for case {}", stageRequest, caseUUIDString);
-            resultStageUUID = caseworkClient.createStage(caseUUID, stageRequest).toString();
-            log.info("Created Stage {} for Case {}", resultStageUUID, caseUUID);
+            stageUUID= UUID.fromString(stageUUIDString);
         }
 
+        CreateCaseworkStageRequest stageRequest = new CreateCaseworkStageRequest(stageTypeString, stageUUID, teamUUID, userUUID, allocationType);
+        log.debug("Creating new stage {} for case {}", stageRequest, caseUUIDString);
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:77]: Are both this log and also log line 64 required? I sort of feel this one would be sufficient?

File: src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java [hunk 2]
```diff
@@ -67,25 +67,17 @@ public String createStage(String caseUUIDString, String stageUUIDString, String
...
+        resultStageUUID = caseworkClient.createStage(caseUUID, stageRequest).toString();
+        log.info("Created Stage {} for Case {}", resultStageUUID, caseUUID);
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:79]: I know that this was probably the original log line - but could we pass through an event from LogEvent here. Helps with identification in OpenSearch

Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:79]: added

File: src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java (hunk 1 above)
Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java:77]: I've removed this line

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**